.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2.1
+
+orbs:
+  ghpr: narrativescience/ghpr@0.0.5
+
+commands:
+  install-deps:
+    description: Install dependencies and initialize pre-commit hooks
+    steps:
+      - run:
+          command: |
+            pip install pre-commit tox flit
+            pre-commit install
+          name: Install dependencies
+
+jobs:
+  test:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - ghpr/build-prospective-branch
+      - install-deps
+      - run: pre-commit run --show-diff-on-failure
+      - run: tox
+      - run: flit build
+      - ghpr/post-pr-comment:
+          comment: Tests failed!
+          when: on_fail
+  publish:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - checkout
+      - run: pip install flit
+      - run: flit build
+      - run: flit publish
+
+workflows:
+  pull_request:
+    jobs:
+      - test:
+          filters:
+            tags:
+              only: /.*/
```

Comment:
Does this tag filter do what we want? Won't this run on merge to master too?

Reply:
Ah, yeah good find. I think it was needed before when they were one workflow but it shouldn't be necessary anymore.